uq.esys.escript.desktop

```
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Terminal=true
+Name=esys escript
+Exec=run-escript
+Type=Application
+Icon=uq.esys.escript.png
```

```suggestion
Icon=uq.esys.escript
```
No need for the extension here.